src/pcre2_substring.c

```diff
@@ -343,7 +343,10 @@ else  /* Matched using pcre2_dfa_match() */
 left = match_data->ovector[stringnumber*2];
 right = match_data->ovector[stringnumber*2+1];
 if (left > match_data->subject_length || right > match_data->subject_length)
+  {
+  PCRE2_DEBUG_UNREACHABLE();
```

Comment:
is this really a debugable failure?, I think this should be reverted and we probably need to add some tests instead

Reply:
Looks like this was already removed, which is fine.